### DISCUSSION TITLE
What i think about spencer

### DISCUSSION THREAD

**Discussion by Dviih** [Milo123459/Spencer#112] [Show and tell]
Spencer is a bot with a cool economy system you can earn money with Voting, a daily reward, gamble and robbing and if you wanna see how a guy is logged in discord computer/moblie spencer has a command to it and we can't forget the fun commands you can remember you dad's joke and see some memes and if you like an ai with less than 5 iq spencer have Invite spencer to your discord server it's a nice bot with a lot of cool features

**Comment by Milo123459** [comment]
Hey, thanks for the review! Just before people start thinking we track you, we don't! Discord API returns it all :)

**Reply by Dviih** [reply]
Lol alright
